packages/fxa-auth-db-mysql/scripts/read-security-events.js

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env node
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * This script takes the uid of a user and will print a security event object of that user
+ * from various tables like securityEvents, sessionTokens and devices tables from fxa database
+ */
+
+// Usage: node read-security-events.js uid
+
+const log = { trace: console.log, error: console.error, stat: console.log, info: console.log } //eslint-disable-line no-console
+const DB = require('../lib/db/mysql')(log, require('../db-server').errors)
+const config = require('../config')
+
+const securityEventsObject = {}
+const scriptInputUid = process.argv[2]
+
+if (! scriptInputUid) {
+  log.error('Error: uid should be passed as an argument to the script!!')
+  process.exit()
+}
+
+const user = {
+  uid: Buffer.from(scriptInputUid, 'hex')
+}
+
+function readEvents() {
+  DB.connect(config).then(async db => {
+    try {
+      const devices = await readDevices(db)
+      securityEventsObject.devices = devices
+
+      const sessions = await readSessionTokens(db)
+      securityEventsObject.sessions = sessions
```

Comment:
I think both devices and sessions can contain buffers, for example in sessions both tokenId and uid are buffers. Can you convert those to hex before printing those to the console? Otherwise it looks like a bunch of gobledygook:

<img width="897" alt="Screenshot 2019-06-10 at 17 06 03" src="https://user-images.githubusercontent.com/848085/59208926-12547b00-8ba2-11e9-8dcb-5dc6de12e012.png">


In addition, can you convert dates from an integer to a human readable format? Perhaps show both. Current output below:

<img width="375" alt="Screenshot 2019-06-10 at 17 06 30" src="https://user-images.githubusercontent.com/848085/59208963-2dbf8600-8ba2-11e9-9451-58f9afcd03d8.png">

Reply:
Done